xapi-stdext-unix-opam.template

```diff
@@ -0,0 +1,2 @@
+depexts: ["linux-headers"] {os-distribution = "alpine"}
+available: [ os = "macos" | os = "linux" ]
```

Comment:
epoll is not available on macOS; not sure if this is relevant here.

Reply:
We could use `iomux` if needed, it supports more OSes. If we want to make `xe` available on more platforms then we should either remove its dependency on stdext, or avoid hard dependencies on epoll.
For now I've updated the template to drop macos.

Reply:
Although contrary to its claim in the package description 'epoll' support isn't actually implement in `iomux` yet, it is just `poll`.